packages/sanity/src/core/releases/tool/components/releaseCTAButtons/ReleaseScheduleButton.tsx

```diff
@@ -99,9 +119,17 @@ export const ReleaseScheduleButton = ({
   const confirmScheduleDialog = useMemo(() => {
     if (status === 'idle') return null
 
+    const _isScheduledDateInPast = isScheduledDateInPast()
+
     return (
       <Dialog
         id="confirm-schedule-dialog"
+        /**
+         * rerenderDialog should force this function to rerun
+         * since the selected scheduled date was in the future when selected
+         * but at time of submit it is in the past
+         */
```

Comment:
I don't understand why we need to re-render the whole calendar though 🤔 or the whole component rather.